src/views/Amalgamation/Information.vue

```diff
@@ -71,7 +72,22 @@
         </template>
       </ExpandableHelp>
 
-      <ResultingBusinessName class="mt-5" />
+      <p
+        v-if="isAmalgamationFilingVertical"
+        class="mt-3">
+        This is a vertical short form amalgamation under section 273 of Business Corporations Act.
+        The amalgamated business will adopt as its notice of articles, the notice of articles of the holding business.
+      </p>
+      <p
+        v-if="isAmalgamationFilingHorizontal"
+        class="mt-3">
+        This is a horizontal short form amalgamation under section 273 of Business Corporations Act.
+        The amalgamated business will adopt as its notice of articles, the notice of articles of the primary business.
+      </p>
```

Comment:
This is a lot of duplication for almost the same text. I suggest adding an inline ternary to display vertical/horizontal and holding/primary instead.